### ISSUE TITLE
ppviewer-1.0.0 failed to build on x86_64

### ISSUE THREAD

**Issue by nysnatuss** [haikuports/haikuports#1980]
[haiku-apps/ppviewer-1.0.0](https://github.com/haikuports/haikuports/blob/master/haiku-apps/ppviewer/ppviewer-1.0.0.recipe) failed to build on x86_64.

Error: [Full log](https://pastebin.com/58RaF4qX)
```
MainWindow.cpp: In constructor 'MainWindow::MainWindow(BRect, const char*, BFile*)':
MainWindow.cpp:69:98: error: taking address of temporary [-fpermissive]
  contentView=new BTextView(rect, "contentView", text_rect,be_fixed_font, &((rgb_color){0,0,0,255}),B_FOLLOW_ALL_SIDES, B_WILL_DRAW);
                                                                                                  ^
MainWindow.cpp: In constructor 'MainWindow::MainWindow(BRect, const char*, const char*, off_t)':
MainWindow.cpp:129:98: error: taking address of temporary [-fpermissive]
  contentView=new BTextView(rect, "contentView", text_rect,be_fixed_font, &((rgb_color){0,0,0,255}),B_FOLLOW_ALL_SIDES, B_WILL_DRAW);
                                                                                                  ^
```

**Comment by Oyami-Srk** [comment]
Fixed.

**Comment by nysnatuss** [comment]
Fixed in #1993.
